build.sbt

```diff
@@ -306,6 +306,11 @@ val settings = Seq(
     name, version, scalaVersion, sbtVersion,
     baseDirectory, datasetDir),
   parallelExecution in Test := false,
+  test in assembly := {},
```

Comment:
If you rebase and squash this should be included in previous changes

Reply:
rebased